Assets/Mirror/Components/InterestManagement/SpatialHashing/SpatialHashingInterestManagement.cs

```diff
@@ -60,6 +60,12 @@ public override void OnRebuildObservers(NetworkIdentity identity, HashSet<Networ
             grid.GetWithNeighbours(current, newObservers);
         }
 
+        [ServerCallback]
```

Comment:
could you move [ServerCallback] to the base function please?

Reply:
None of the methods in the abstract base InterestManagement class are decorated...all the derived classes have the decorations, so I followed that pattern.  If you want the base class methods decorated, that's a separate PR.

Merge this and I'll do that after.